autogluon/task/tabular_prediction/presets_configs.py

```diff
@@ -0,0 +1,68 @@
+
+
+def unpack(g):
+    def _unpack_inner(f):
+        def _call(**kwargs):
+            return f(**g(**kwargs))
+        return _call
+    return _unpack_inner
+
+
+# Dictionary of preset fit() parameter configurations.
+preset_dict = dict(
+    # Best quality with little consideration to inference time or disk usage. Achieve even better results by specifying a large time_limits value.
```

Comment:
consider replacing "quality" -> "predictive accuracy" in these comments to be more clear

Reply:
Updated